src/scss/components/pages.scss

```diff
@@ -79,16 +79,38 @@
 
 // Page sidebar
 .page-sidebar-header {
-  border-bottom: 1px solid $gray-600;
-  padding-bottom: 8px;
+  border-bottom: 1px solid $sidebar-bottom-border-color;
+  padding-bottom: 15px;
   margin-top: 4px;
+  margin-bottom: 1.8rem;
```

Comment:
We have a mix of REMs and pxs here. I'd recommend using REMs so they all adjust with base font size - for now we can just update this file.

Reply:
Done!